addons/base_automation/views/ir_actions_server_views.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <!-- Automation: Server Action Form View -->
-    <record id="view_server_action_form" model="ir.ui.view">
-        <field name="name">base.automation.server.action.form</field>
+    <record id="ir_actions_server_view_form_automation" model="ir.ui.view">
```

Comment:
Note that we don't put the overridden module name in the id usually, only in the field name below, as it prefixes with the module name automatically with a ".", correct?

Reply:
But it is a different view (primary) so in this case adding the suffix allows to know its usage.